projects/aca-shared/src/lib/services/app.service.ts

```diff
@@ -65,6 +65,9 @@ export class AppService implements OnDestroy {
   private ready: BehaviorSubject<boolean>;
   ready$: Observable<boolean>;
 
+  public hideSidenav = new BehaviorSubject<boolean>(false);
+  cast = this.hideSidenav.asObservable();
```

Comment:
why it is called "cast" and why "cast" is responsible for the hiding the side nav? you probably need making "hideSidenav" private and making this one public and called 'hideSidenav$'